src/pages/Information/Information.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+const Information = () => (
+  <>
+    <div data-test="information-display">
+      <h2>What is HacktoberFest?</h2>
```

Comment:
♻️  Could you add tests for these headings?

Reply:
Modified the title and added tests for each tag.